cryptol-remote-api/python/pyproject.toml

```diff
@@ -14,10 +14,12 @@ include = [
 
 [tool.poetry.dependencies]
 python = ">=3.8.0,<4"
-requests = "^2.31.0"
+requests = "<2.31.0.20240310"
 BitVector = "^3.4.9"
 argo-client = "0.0.11"
 typing-extensions = "^4.1.1"
+urllib3 = "<2.2.0"
+types-requests = "<2.31.0.20240310"
```

Comment:
Leave a comment above these two dependencies stating that while the code itself doesn't directly depend on them, `mypy` _does_ depend on them, so we include them anyway to ensure that their versions are pinned. Worth citing https://github.com/python/mypy/issues/10600#issuecomment-857690947 as inspiration for this trick.

Reply:
Added in most recent version.